### PR TITLE
Adds attack verbs for heart and tongue

### PR DESCRIPTION
**Pull request by nullbear**
URGENT

:cl: nullbear
rscadd: Added attack verbs for heart and tongue organs
/:cl:

### CODE REVIEW

**Comment by Incoming5643** [comment]
:+1: but not everything needs a changelog

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -253,6 +254,7 @@
 	zone = "mouth"
 	slot = "tongue"
 	var/say_mod = null
+	attack_verb = list("licked", "slobbered", "slapped")
```
Comment (KazeEspada) [inline code/modules/surgery/organs/organ_internal.dm]: don't forget "frenched" and "tongued"

**Comment by coiax** [comment]
Remove the quotes on the changelog, they're not needed

File: code/modules/surgery/organs/organ_internal.dm (hunk 1 above)
Comment (nullbear) [inline code/modules/surgery/organs/organ_internal.dm]: Added

**Comment by optimumtact** [comment]
having fun?

**Comment by Niknakflak** [comment]
I enjoy the pun of the heart

**Comment by Durkel** [comment]
Looks like it's time for a tongue lashing.

**Pull request merged**